src/config.rs

```diff
@@ -231,6 +238,33 @@ where
     deserializer.deserialize_str(ColorVisitor)
 }
 
+fn deserialize_align<'de, D>(deserializer: D) -> Result<Alignment, D::Error>
```

Comment:
It would be better to use serde's derive macros as described [here](https://serde.rs/remote-derive.html).